release-notes/release-3.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: February 28, 2024
+---
+
+# Features
+- Metrics Explorer in Settings provides users a single location to view all of their collected metrics -  users can search for metrics by resource name, key, or label names
+- Dynamic filters in the alert module for users based on their existing alert configurations
+- Users can mute their already created Alerts in the details screen
+- Alerts are dynamically changed based on the last triggered alert for that rule
+- Data Ingestion Intervals allow users to control the interval on which they ingest data for any new dashboard
+- Kafka Integration
+
+# Changes
+- Increased trace ingestion capabilities (200,000 traces/min) to decrease probability of crashes
+- At least one ingestion pipeline will be active from the Ingestion Controls window in Settings to ensure data ingestion
```

Comment:
This one could be split out into a few:
1. Change: from Advanced -> Ingestion controls
2. New: Expanded Ingestion Controls to include Traces and Metrics
3. New: Kuberentes level ingestion controls